spec/struct.dd

```diff
@@ -512,7 +512,7 @@ $(SECTION3 $(LEGACY_LNAME2 AssignOverload, assign-overload, Identity Assignment
         $(P where $(CODE opAssign) is a member function of S:)
 
         ---
-        ref S opAssign(S s)
+        ref S opAssign(ref S s)
```

Comment:
This is incorrect change. The assigned lvalue needs to be copied before the assignment.